web/src/pages/UpdatePasswordPage/UpdatePasswordPage.js

```diff
@@ -0,0 +1,78 @@
+import { routes, navigate } from '@redwoodjs/router'
+import { useAuth } from '@redwoodjs/auth'
+import { Form, Submit } from '@redwoodjs/forms'
+import { useFlash } from '@redwoodjs/web'
+
+import InputTextForm from 'src/components/InputTextForm'
+import MainLayout from 'src/layouts/MainLayout'
+import Seo from 'src/components/Seo/Seo'
+
+const UpdatePasswordPage = () => {
+  const { addMessage } = useFlash()
+  const { client } = useAuth()
+  const onSubmit = ({ password, confirm }) => {
+    if (password !== confirm || !password) {
+      addMessage("Passwords don't match, try again", {
+        classes: 'bg-red-300 text-red-900',
+      })
+      return
+    }
+    client
+      .currentUser()
+      .update({ password })
```

Comment:
This update can be used to update any property on the user. And while I haven't built email update UI, if you plan to it's worth knowing there is an [undocumented token needed](https://github.com/netlify/gotrue-js/issues/61)